tests/scripts/set_psa_test_dependencies.py

```diff
@@ -22,6 +22,23 @@
 import re
 import sys
 
+def is_systematic_dependency(dep):
+    """Whether dep is a PSA dependency which is determined systematically."""
+    return dep.startswith('PSA_WANT_')
+
+def dependencies_of_symbol(symbol):
```

Comment:
This function returns a set. Currently the set contains at most 1 element because the configuration mechanism corresponds closely to the symbols by design, but this might not be true if the script starts handling more complex cases. Furthermore the function can return a set with 0 elements, and 0 is plural in English.